ansible_collections/arista/avd/molecule/anta_runner/anta/reports/filtered_run/anta_report.json

```
@@ -307,6 +307,28 @@
         "messages": [],
         "custom_field": "Generated by AVD"
     },
+    {
+        "name": "dc1-leaf1a",
+        "test": "VerifyDNSLookup",
+        "categories": [
+            "services"
+        ],
+        "description": "Verifies the DNS name to IP address resolution.",
```

Something seems off here, but someone with better English skills should take a look :)
```suggestion
        "description": "Verifies DNS name to IP address resolution.",
```

---

:) We'll find something better and fix it on the ANTA side.